config/gatsby-remark-embedder/custom-yt-embedder.js

```diff
@@ -0,0 +1,59 @@
+const shouldTransform = url => {
+  const { host, pathname, searchParams } = new URL(url)
+
+  return (
+    host === 'youtu.be' ||
+    (['youtube.com', 'www.youtube.com'].includes(host) &&
+      pathname.includes('/watch') &&
+      Boolean(searchParams.get('v')))
+  )
```

Comment:
Why not add `'youtu.be'` to the `.includes` array?

Reply:
I copied this code over from `gatsby-remark-embed` so I'm not sure why the code is written like that. I'll update it and test if it works the same :)

Reply:
I also noticed and checked. I think it's due to the URL difference 
- `https://youtu.be/kLKBcPonMYw`
- `https://www.youtube.com/watch?v=kLKBcPonMYw`

Reply:
Updated the code and everything looks to be working!

Reply:
Good to know, I read the parentheses wrong, sorry for the false suggestion!